cloudformation.properties

```diff
@@ -2,12 +2,18 @@ name=Cloudformation
 category=External Analysers
 descripton=Cloudformation rules for json and yaml, integrates cfn-nag reports.
 homepageUrl=https://github.com/Hack23/sonar-cloudformation-plugin
-archivedVersions=2.0.1,2.0.2,2.0.3,2.0.4,2.0.5,2.0.7,2.0.8,2.0.9
-publicVersions=2.0.10
+archivedVersions=2.0.1,2.0.2,2.0.3,2.0.4,2.0.5,2.0.7,2.0.8,2.0.9,2.1.18
+publicVersions=2.1.18
 
 defaults.mavenGroupId=com.hack23.sonar
 defaults.mavenArtifactId=sonar-cloudformation-plugin
 
+2.1.18.description=Adds support for checkov. Include rules for terraform, kubernetes and azure templates.
+2.1.18.date=2021-11-09
+2.1.18.sqVersions=[7.9,8.9.*]
+2.1.18.changelogUrl=https://github.com/Hack23/sonar-cloudformation-plugin/releases/tag/sonar-cloudformation-plugin-2.1.18
```

Comment:
404 for me

Reply:
Release I made an error, the release was 2.1.8 and not 2.1.18 so did some cut and paste errors. But corrected but links now.